pkg/controllers/provisioning/v1alpha1/allocator/controller.go

```diff
@@ -49,6 +49,10 @@ func (c *Controller) Interval() time.Duration {
 	return 5 * time.Second
 }
 
+func (c *Controller) Name() string {
+	return "Allocator"
```

Comment:
Is the syntax of this in line with other controller names? Just want to make sure it looks reasonable. We could also do something like Provisioner/Allocator. Just make sure the prometheus metric names all line up as expected.

Reply:
Ah yeah. They're lowercase. I'm a fan of doing the / notation.